src/historyevent.ts

```diff
@@ -2,14 +2,19 @@ import { HistoryEventType } from "./classes";
 
 export class HistoryEvent {
     constructor(
```

Comment:
Same comment here on constructors having lots of arguments - makes reading code really hard if there is more than a few arguments.

Reply:
This is true, and it bothered me while I was writing it (but I shrugged it off.) I should learn to be less tolerant of coding annoyances. 🤔